### PR TITLE
[wip]fixup single paren, pod overview

### PR DESCRIPTION
**Pull request by kbhawkey**
- Fix single parenthesis in `What's next` section
- Other cleanup (duplicated text, use of tooltips)
- Moved "Storage" and "Networking" sections under "Using Pods" section.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 1]
```diff
@@ -182,20 +180,15 @@ template, the StatefulSet starts to create new Pods based on the updated templat
...
 directly observe or manage any of the details around pod templates and updates; those
 details are abstracted away. That abstraction and separation of concerns simplifies
 system semantics, and makes it feasible to extend the cluster's behavior without
 changing existing code.
 
-## Resource sharing and communication
-
-Pods enable data sharing and communication among their constituent
-containers.
-
```
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/_index.md:198]: If we remove the  “Resource sharing and communication” heading, then should this become a 2nd level heading, along with Pod networking? Otherwise, these become grouped under “Working with Pods”, which doesn't seem right.

Two more options:
- move “Storage in Pods” and “Pod networking” to come immediately after “How Pods manage multiple containers”
- move “How Pods manage multiple containers” down to here.

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/pods/_index.md:198]: Thanks for reviewing. Will look at this again.

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/pods/_index.md:198]: I think it makes sense to follow the "Using Pods" main section with the 
"How Pods manage multiple containers", Storage, and Networking sections.
Let me know if you have other concerns.

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 2]
```diff
@@ -105,10 +105,41 @@ Some Pods have {{< glossary_tooltip text="init containers" term_id="init-contain
 Pods natively provide two kinds of shared resources for their constituent containers:
 [networking](#pod-networking) and [storage](#pod-storage).
 
+### Storage in Pods {#pod-storage}
+
```
Comment (kbhawkey) [inline content/en/docs/concepts/workloads/pods/_index.md:109]: Is this relocation an improvement? If not, I am happy to revert this change.

Comment (tengqm) [reply content/en/docs/concepts/workloads/pods/_index.md:109]: To be honest, I fail to see the improvement. In the original flow, I can see that both storage and network are used to facilitate communication. Not a strong opinion here though.

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/pods/_index.md:109]: I initially made this change because the original text and heading seemed more closely related to the content found in the
"Using Pods" section (Pods natively provide two kinds of shared resources for their constituent containers: networking and storage).
From https://kubernetes.io/docs/concepts/workloads/pods/#resource-sharing-and-communication,
`Pods enable data sharing and communication among their constituent containers.`

Comment (tengqm) [reply content/en/docs/concepts/workloads/pods/_index.md:109]: Yes. That is why we have the Pod abstraction. I'm okay with the reorg, though the improvement is not evident to me.
As a non-native speaker, I'm not sure I understand the difference between "Using Pods" and "Working with Pods". So probably this page needs more work in the future.

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/pods/_index.md:109]: Yes, I agree that the headings, "Using Pods" and "Working with Pods", are similar.

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/pods/_index.md:109]: I'll look again at the page.
